packages/graphile-build-pg/src/plugins/PgTypesPlugin.js

```diff
@@ -255,7 +255,7 @@ export default (function PgTypesPlugin(
         sql.fragment`(${fragment})::numeric::text`;
       const pgTweaksByTypeIdAndModifer = {};
       const pgTweaksByTypeId = Object.assign(
-        // ::text rawTypes
+        // '::text' rawTypes
```

Comment:
This is to work around a bug in prettier: https://github.com/prettier/prettier/issues/6102